setup.cake

```diff
@@ -1,7 +1,7 @@
 #tool nuget:?package=GitVersion.CommandLine
 #tool nuget:?package=OpenCover&version=4.6.519
 #tool nuget:?package=Codecov&version=1.0.3
-#addin nuget:?package=Cake.Figlet&version=1.0.0
+#addin nuget:?package=Cake.Figlet
```

Comment:
Normally I would say no to removing the version from addins.
But since I'll be converting the build script to use Cake.Recipe it's okay this time.